### PR TITLE
Critical Support Doc Updates

### PR DESCRIPTION
**Pull request by itsalyse**
This PR covers the second round of critical support doc updates.

Email hosting support - https://docs.google.com/document/d/1BOqm4JsSFOBArZrcpqKjrwhmFVIHdvg4pKW8WMG-A6s/edit#heading=h.2yes2wlb19pp

Web hosting - https://docs.google.com/document/d/1OfH1Bpe2VkXg7HfV6cPUxnLxUAB0KMER-qsw3YBQNJk/edit

Payment methods - https://docs.google.com/document/d/1uOrCeHv8rBuD-l6woCxWHUlVgk4mm43bpiu2OiNj2Lw/edit

Understanding your invoice - https://docs.google.com/document/d/1fZKYuh9MYCRguHYpcNyvhm6xMByH3Yw1JujQTX-7oEs/edit

Related to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/understanding-invoice.md [hunk 1]
```diff
@@ -15,31 +15,51 @@ categories:
...
+
+|  |**Name**|**Description**|
+|:--|:--|:--|
+| 1 |  **Invoice Number** | The unique number assigned to your invoice. |
+| 2 | **Invoice Date** | The date the invoice was created. The invoice date is the first day of your billing cycle. For example, if your billing cycle is April 20 - May 20, your Invoice Date is April 20. |
+| 3 |  **Terms**     | Indicates whether your invoice is `Due Upon Receipt`, `Open`, or `Paid`. |
+| 4 |  **Status** | Indicates whether or not your invoice has been paid.  |
+| 5 | **To**  | Your company name and address. To update this information, see Billing Settings. |                        
+
+ **NEW SCREENSHOT**
```
Comment (itsalyse) [inline content/articles/understanding-invoice.md:49]: https://drive.google.com/drive/u/3/folders/18XIhbV9BE8RtMf-mTbfmxM2b72hqSCTu

File: content/articles/understanding-invoice.md [hunk 2]
```diff
@@ -15,31 +15,51 @@ categories:
...
-| 7. | **Subtotal**       | The total amount for the invoice before any credits or discounts.                                                                                                                     |
-| 8. | **Credit**         | The amount of any credit deducted from your [account credit balance](/articles/account-subscription-balance/).                                                                        |
-| 9. | **Total**          | The amount charged to your credit card for the invoice billing cycle.                                                                                                                                         |
+If you have multiple high-volume zones or receive a very high number of monthly queries, [contact sales](https://dnsimple.com/sales) for Enterprise pricing, and we'll get you set up with what you need.
 
+## Explanation of your invoice
+
+Here's a closer look at what your invoice means:
+
+**NEW SCREENSHOT**
```
Comment (itsalyse) [inline content/articles/understanding-invoice.md:39]: https://drive.google.com/drive/u/3/folders/18XIhbV9BE8RtMf-mTbfmxM2b72hqSCTu

Comment (itsalyse) [reply content/articles/understanding-invoice.md:39]: @dperrymorrow could you please add this screenshot and the one above?

**Review by stephanoodle — APPROVED** [review]

**Review by stephanoodle — APPROVED** [review]

**Review by dperrymorrow — APPROVED** [review]
I have added in the screenshots and also removed trailing whitespace characters. Thanks!

**Review by dperrymorrow — APPROVED** [review]

**Review by stephanoodle — APPROVED** [review]

**Pull request merged**